### PR TITLE
Changes on dlgPermuteColumn

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  there is still something i need to discuss with you. Is the default values for seed and Number of Permutation Columns be one? 
I have realised that if i have more than one Number of Permutation Columns doesnt work nicely. Have a look at this and see how i can get it better.

**Comment by dannyparsons** [comment]
Number of Permutation Columns has default 1.
Seed doesn't have a default value in R. If we have to have one for the numeric up/down, then I suggest 0 for now.

Yes writing multiple columns does not work well generally. I need to look into that so you shouldn't worry about it for your dialog right now.

**Comment by maxwellfundi** [comment]
@dannyParsons is this okay for merging?

**Comment by dannyparsons** [comment]
I don't see a NameChanged event for your input control so I don't see how this is working without this?

**Comment by maxwellfundi** [comment]
@dannyparsons  I was wondering if this is ready to be merged or what more should i have to do on it

File: instat/dlgPermuteColumn.vb [hunk 1]
```diff
@@ -21,20 +21,28 @@ Public Class dlgPermuteColumn
...
     Private Sub SetDefaults()
+        ucrReceiverPermuteRows.Selector = ucrPermuteRowsSelector
+        ucrReceiverPermuteRows.SetMeAsReceiver()
+        ucrInputPermuteRows.SetPrefix("permute")
         chkSetSeed.Checked = False
         nudSetSeed.Visible = False
         TestOkEnabled()
     End Sub
+    Private Sub ReopenDialog()
+        ucrInputPermuteRows.SetPrefix("permute")
```
Comment (dannyparsons) [inline instat/dlgPermuteColumn.vb]: I fixed this for the Input control so this line can be removed now.

File: instat/dlgPermuteColumn.vb [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    Private Sub InitialiseDialog()
+        ucrReceiverPermuteRows.Selector = ucrPermuteRowsSelector
+        ucrReceiverPermuteRows.SetMeAsReceiver()
+        clsSetSeedFunc.SetRCommand("set.seed")
+        ucrBase.clsRsyntax.SetFunction("replicate")
+        ucrBase.clsRsyntax.AddParameter("expr", clsRFunctionParameter:=clsSetSampleFunc)
+        clsSetSampleFunc.SetRCommand("sample")
+        clsSetSampleFunc.AddParameter("x", clsRFunctionParameter:=ucrReceiverPermuteRows.GetVariables())
+        clsSetSampleFunc.AddParameter("replace", "FALSE")
+        clsSetSampleFunc.AddParameter("size", ucrPermuteRowsSelector.ucrAvailableDataFrames.iDataFrameLength)
```
Comment (dannyparsons) [inline instat/dlgPermuteColumn.vb]: This line should be run under the DataFrameChanged event and not here.

**Comment by maxwellfundi** [comment]
@dannyparsons this is now fixed

**Pull request merged**